test/e2e/mapml-viewer/mapml-viewerAttributes.test.js

```diff
@@ -0,0 +1,27 @@
+import { test, expect, chromium } from '@playwright/test';
+
+test.describe("Adding Width and Height Attribute to mapml-viewer", () => {
+  let page;
+  let context;
+  test.beforeAll(async function() {
+    context = await chromium.launchPersistentContext('');
+    page = context.pages().find((page) => page.url() === 'about:blank') || await context.newPage();
+    await page.goto("mapml-viewerAttributes.html");
+  });
+  test.afterAll(async function () {
+    await context.close();
+  });
+
+  test("Setting Width and Height Attibute to mapml-viewer", async () => {
+    let height_attribute_value = await page.$eval("body > mapml-viewer",(val) => val.getAttribute("height"));
+    let width_attribute_value = await page.$eval("body > mapml-viewer",(val) => val.getAttribute("width"));
+    let height = await page.$eval("body > mapml-viewer", (val) => val._container.style.height);
+    let width = await page.$eval("body > mapml-viewer", (val) => val._container.style.width);
+    expect(height).toEqual(height_attribute_value+"px");
```

Comment:
This is good.  You could go so far as to name the "width" (and height) variable like "width_with_px" or "[used_width](https://developer.mozilla.org/en-US/docs/Web/CSS/used_value)", with a comment link to used values.  Tests are all about what you mean, so being a bit more verbose, but especially explicit, never hurts.